### PR TITLE
Added link to the Wikipedia page of Assignment problem

### PR DESCRIPTION
**Pull request by asdine**

### CODE REVIEW

**Comment by skateinmars** [comment]
Maybe we should also link to the C implementation that this was based on?

**Comment by Nyks06** [comment]
Nice idea, 👍

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 [![GoDoc](https://godoc.org/github.com/heetch/lapjv?status.svg)](https://godoc.org/github.com/heetch/lapjv)
 
-This repository is the Golang implementation of the **LAPJV** algorithm and includes utilitaries to facilitate the first usage and tests of the library.
+This repository is the Golang implementation of the [**LAPJV**](https://en.wikipedia.org/wiki/Assignment_problem) algorithm and includes utilitaries to facilitate the first usage and tests of the library.
```
Comment (arkan) [inline README.md]: `utilitaries`: do you mean tools?

Comment (arkan) [inline README.md]: I prefer `tools` but your call as long as the typo is fixed :)

Comment (Nyks06) [inline README.md]: fixed

**Comment by asdine** [comment]
🆙

**Comment by arkan** [comment]
👍

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 [![GoDoc](https://godoc.org/github.com/heetch/lapjv?status.svg)](https://godoc.org/github.com/heetch/lapjv)
 
-This repository is the Golang implementation of the **LAPJV** algorithm and includes utilitaries to facilitate the first usage and tests of the library.
+This repository is the Golang implementation of the [LAPJV](http://www.magiclogic.com/assignment.html) algorithm for [assignment problem](https://en.wikipedia.org/wiki/Assignment_problem) and includes tools to facilitate the first usage and tests of the library.
```
Comment (jhchabran) [inline README.md]: It can be rephrased a bit to make it simpler:

> to ease library usage and testing

**Comment by skateinmars** [comment]
:+1: can the person that will merge please squash the commits?

**Pull request merged**